### DISCUSSION TITLE
Completing words from different sources Ctrl-N

### DISCUSSION THREAD

**Discussion by chiefjester** [hrsh7th/nvim-cmp#2014] [Q&A]
I was wondering if I should file a bug, it seems that using Ctrl-N/Ctrl-P only suggest current buffer, but not other.

I tried to use Ctrl-X / Ctrl-L so it does read other buffer, how can I disable cmp from using Ctrl-N and Ctrl-P?

**Comment by fitrh** [comment]
You con show suggestion from all buffers with

```lua
cmp.setup({
  sources = {
    {
      name = 'buffer',
      option = {
        get_bufnrs = function()
          return vim.api.nvim_list_bufs()
        end
      },
    },
  },
})
 ```
